### PR TITLE
Ec Collect Response

### PR DESCRIPTION
**Pull request by ericthewry**
@sf-interns/owners 
Hey guys, pull this down, test it out.  Lemme know if it doesnt work.

Design/style feedback is greatly appreciated as well

### CODE REVIEW

**Comment by ericthewry** [comment]
@sf-interns/owners 
Feel free to check this out.

File: app/services/SolutionService.coffee [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+class SolutionService
+
+  constructor: (@board, goals) ->
+    @goals = []
+    for g in goals
+      @goals.push g
+
+  isSolution: (clickedCells) ->
+    return false unless clickedCells? and clickedCells.length >=3
```
Comment (bmwwoo) [inline app/services/SolutionService.coffee]: >= 3 (space)

We can fix these small things in a reorganizing getting ready for GH Pages PR

**Comment by bmwwoo** [comment]
LGTM :+1:

File: app/views/Board.coffee [hunk 2]
```diff
@@ -45,17 +48,18 @@ class Board
...
       @cells.push []
       for col in [0...@dimension]
-        cell = new @Cell col, row, width, @two, this, @clickHandler, symbols[@toIdx @boardValues[row][col]]
+        cell = new @Cell col, row, width, @two, this, @clickHandler, @symbols[@toIdx @boardValues[row][col]]
         cell.setColor @Colors.cell
         cell.setBorder @Colors.cellBorder
         @cells[row].push cell
 
   deleteCells: (solution) ->
+    console.log 'delete cells', solution
```
Comment (shubhamgogna) [inline app/views/Board.coffee:62]: Logging to console here should probably be removed.

Comment (bmwwoo) [inline app/views/Board.coffee:62]: I think we'll remove all the console.logs and tests in a later PR but good catch

**Pull request merged**